assets/sass/modules/_enroll.scss

```diff
@@ -403,6 +403,7 @@
 [name="sharedSecret"] {
   text-align: center;
   cursor: text;
+  opacity: 1;
```

Comment:
This input is being located inside the span with class .o-form-disabled, which has a property opacity: 0.4. For some reason Firefox applied this opacity once again to this input which is direct sibling, so we have input with opacity 0.4 too. As a result input was too light. Now it looks fine.